man/man8/zpool.8

```diff
@@ -1635,10 +1635,12 @@ with no flags on the relevant target devices.
 .Oo Oo Ar pool Ns ... Oc Ns | Ns Oo Ar pool vdev Ns ... Oc Ns | Ns Oo Ar vdev Ns ... Oc Oc
 .Op Ar interval Op Ar count
 .Xc
-Displays I/O statistics for the given pools/vdevs. You can pass in a
-list of pools, a pool and list of vdevs in that pool, or a list of any
-vdevs from any pool. If no items are specified, statistics for every
-pool in the system are shown.
+Displays I/O statistics for the given pools/vdevs; this does not reflect
+physical I/O, but rather how ZFS submitted requests to the block layer which
+will merge them into larger operations, if possible.
```

Comment:
I'm not sure that "block layer" is the most helpful term here.  It's easily confused for the layer below ZFS.  But I think what you're talking about is the lower level vdev / vdev queue / aggregation layer.  You may have been thrown off by @behlendorf 's comment (https://github.com/zfsonlinux/zfs/issues/2888#issuecomment-62823972) which is talking about a different potential problem from what was reported in the Issue.

We might want to have a more elaborate comment here to give some examples of things that cause different number of operations at each level.  And also to distinguish between behavior that impacts number of i/os, vs that which impacts number of bytes (bandwidth).

Maybe something like this:
`Displays I/O statistics for the given pools/vdevs.  This reflects logical i/o counts.  For physical i/o's, see iostat(1).  Nearby logical i/os may be aggregated (combined) into a single i/o.  Additional logical i/os may be generated to write mirror or parity data. ...`

Reply:
```
     zpool iostat [[[-c SCRIPT] [-lq]]|-rw] [-T u|d] [-ghHLnpPvy] [[pool...]|[pool vdev...]|[vdev...]] [interval [count]]
             Displays logical I/O statistics for the given pools/vdevs. Physical I/Os may be observed via iostat(1).  If writes are located nearby, they may be merged into a single larger operation.
             Additional I/O may be generated depending on the level of vdev redundancy.  To filter output, you may pass in a list of pools, a pool and list of vdevs in that pool, or a list of any
             vdevs from any pool. If no items are specified, statistics for every pool in the system are shown.  When given an interval, the statistics are printed every interval seconds until ^C is
             pressed. If -n flag is specified the headers are displayed only once, otherwise they are displayed periodically. If count is specified, the command exits after count reports are printed.
             The first report printed is always the statistics since boot regardless of whether interval and count are passed. However, this behavior can be suppressed with the -y flag. Also note that
             the units of K, M, G ... that are printed in the report are in base 1024. To get the raw values, use the -p flag.

```

how's that?